### ISSUE TITLE
Broken Links!!

### ISSUE THREAD

**Issue by joslloand** [ambisonictoolkit/atk-sc3#105]
Broken links found in v5.0.3

```
WARNING: SCDoc: In ~/Documents/Developer/atk-sc3/HelpSource/Tutorials/ABCs-of-the-ATK.schelp
  Broken link: 'Tutorials/ATK-FAQs#How%20do%20I%20report%20a%20bug?#How do I report a bug?'
WARNING: SCDoc: In ~/Documents/Developer/atk-sc3/HelpSource/Tutorials/ABCs-of-the-ATK.schelp
  Broken link: 'Tutorials/ATK-FAQs#What%20is%20the%20highest%20order%20supported%20for%20spherical%20decomposition?#What is the highest order supported for spherical decomposition?'
```

**Comment by joslloand** [comment]
... just two links...

**Comment by joslloand** [comment]
Fixed with c87287ff9f2b25d4c626399461de285bd4afeef1, available in next release (v5.0.4)
